packages/api/src/routes/internal/medical/patient.ts

```diff
@@ -1170,4 +1171,33 @@ router.post(
   })
 );
 
+/** ---------------------------------------------------------------------------
+ * PUT /internal/patient/:id/external-data
+ *
+ * Resets the external data for a specific patient and HIE source by removing it completely.
+ *
+ * @param req.params.id The patient ID.
+ * @param req.query.source The HIE source (COMMONWELL or CAREQUALITY).
+ * @param req.query.cxId The customer ID.
+ * @return 200 OK upon successful reset.
+ */
+router.put(
```

Comment:
`delete`? this is essentially removing the external data for a specific HIE

Reply:
+1

Reply:
I had it as a delete but thought since we are updating the externalData technially that it should be a put but I was on the fence anyways so ill change back to delete